components/Dropdown/dropdown.js

```diff
@@ -0,0 +1,17 @@
+let dropdown = document.querySelector(".header__dropdown");
+
+class Dropdown {
```

Comment:
Nice job on this dropdown! I hope Dan's solution along with the lecture helped you understand this part. Although dropdown itself is not so important, it is a neat thing to know.